app/assets/stylesheets/homepage.css.scss

```diff
@@ -190,49 +195,48 @@ margin-left: 5%;
 }
 
 .button-text {
-    font-size: 36px;
+  font-size: 36px;
 }
 
 .plan-trip-question {
-    font-size: 24px;
-    font-weight: 700;
+  font-size: 24px;
+  font-weight: 700;
 }
 
 
 .home-mission {
-    text-align: center;
-    height: 400px;
-    margin-top: 8%;
+  text-align: center;
+  margin-top: 8%;
+  margin-bottom: 45px;
 }
 
 .home-mission p {
-    text-align: center;
-    font-size: 24px;
-    font-weight: 200;
-    padding: 75px;
-    color: white;
-    text-shadow: black 2px 2px 2px;
+  text-align: center;
+  font-size: 1.5em;
+  line-height: 1.5em;
+  padding: 55px 0 75px 0;
+  background-color: rgba(255, 255, 255, 0.6);
+  background: rgba(255, 255, 255, 0.6);
 }
 
 .home-about-us {
-    text-align: center;
-    height: 400px;
-
+  text-align: center;
+  height: 400px;
 }
 
 .home-trip-reviews {
-    text-align: center;
-    height: 400px;
+  text-align: center;
+  height: 400px;
 }
 
 .bubble h4 {
-    margin: 0px;
-    padding: 1%;
-    font-family: 'Yellowtail', cursive;
-    font-size: 36px;
-    font-weight: 300;
-    background-color: rgba(255,255,255,.3);
-    text-shadow: #c8a95b 3px 1px 1px;
+  margin: 0px;
+  padding: 1%;
+  font-family: 'Yellowtail', cursive;
+  font-size: 36px;
+  font-weight: 300;
+  background-color: rgba(255,255,255,.3);
```

Comment:
`.3` should be written with a leading zero as `0.3`<br>Commas in function arguments should be followed by a single space